operator-common/src/main/java/io/strimzi/operator/common/metrics/CaCertificateMetricKey.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.metrics;
+
+import java.util.Locale;
+
+/**
+ * Metric key for metrics specified by kind, namespace and cluster name.
+ */
+public class CaCertificateMetricKey extends MetricKey {
```

Comment:
As mentioned in the previous comment ... I would call this `CertificateMetricKey`, `CaType` just `Type` and the values `CLUSTER_CA` and `CLIENTS_CA`.